docs/en/observability/synthetics-settings.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[synthetics-settings]]
+= Adjust Synthetics settings
+
+++++
+<titleabbrev>Adjust settings</titleabbrev>
+++++
+
+There are several Synthetics settings you can adjust in {kib}.
+
+[discrete]
+[[synthetics-settings-alerting]]
+= Alerting
+
+Alerting enables you to detect complex conditions using *rules* across Observability apps
+and send a notification using *connectors*.
+
+When you create a new synthetic monitor, a new default synthetics rule will be applied.
+To edit the default rule:
+
+. Click *Alerts and rules* in the top bar.
+. Click *Monitor status rule* to open a panel where you can edit the rule's configuration.
+
+To view all existing rules or create new rule:
+
+. Click *Alerts and rules* in the top bar.
+. Click *Manage rules*.
+
+Read more about creating new rules in <<create-alerts-rules>>.
+
+In the *Alerting* tab on the Synthetics Settings page, you can add and configure connectors.
+Read more about all available connectors in <<action-types-duration>>.
```

Comment:
Might also be worth adding something like _"If you are running in Elastic Cloud, then an SMTP connector will automatically be configured, allowing you to easily set up email alerts"_